docusaurus/docs/create-a-plugin/extend-a-plugin/add-authentication-for-data-source-plugins.md

```diff
@@ -148,11 +148,11 @@ To forward requests through the Grafana proxy, you need to configure one or more
    ]
    ```
 
-  :::note
+:::note
```

Comment:
I think this still might be broken as it resets the numbered list

Reply:
Sorry was looking at the wrong bit... this is fine, its further up in https://grafana-dev.com/developers/plugin-tools/create-a-plugin/extend-a-plugin/add-authentication-for-data-source-plugins#add-secret-configuration-to-your-data-source-plugin which is broken and could be fixed

Reply:
Ah well spotted! I'll fix that shortly. 👍

Reply:
Having jumped into this I've discovered this is a bit more widespread than I first realised. As it's the same on prod I'm gonna open a follow up PR that will address all these broken indentations. Thanks for highlighting it @sympatheticmoose